packages/checkout-com/src/useCko.ts

```diff
@@ -20,26 +22,36 @@ interface PaymentMethodsConfig {
   paypal?: any;
 }
 
-enum PaymentMethod {
-  NOT_SELECTED = 0,
-  CARD,
-  SAVED_CARD,
-  KLARNA,
-  PAYPAL
-}
+const selectedPaymentMethod = ref(CKO_PAYMENT_TYPE.NOT_SELECTED);
 
-const selectedPaymentMethod = ref(PaymentMethod.NOT_SELECTED);
+const setSavePaymentInstrument = (newSavePaymentInstrument: boolean) => {
+  localStorage.setItem(getSaveInstrumentKey(), JSON.stringify(newSavePaymentInstrument));
+};
+const loadSavePaymentInstrument = () => (localStorage.getItem(getSaveInstrumentKey())
+  ? JSON.parse(localStorage.getItem(getSaveInstrumentKey()))
```

Comment:
Forcing a oneliner here is not the best approach, especially when you load something from the storage.
Assign the result to variable and then check it to avoid loading it again.